pkg/stanza/operator/transformer/recombine/transformer_test.go

```diff
@@ -282,6 +282,52 @@ func TestTransformer(t *testing.T) {
 				entryWithBody(t1, map[string]any{"message": "Another log message"}),
 			},
 		},
+		{
+			"CombineSplitedUnicode",
```

Comment:
split - split - split https://english.stackexchange.com/questions/467873/the-past-participle-of-split-split-or-splitted
```suggestion
			"CombineSplitUnicode",
```